bot.rb

```diff
@@ -87,16 +87,14 @@ def process_closed_issue(issue)
     end
 
     def process_open_pr(pr, needs_attention_prs)
-      days_since_updated = (Time.now - pr.updated_at) / 60.0 / 60.0 / 24.0
+      days_since_created = (Time.now - pr.created_at) / 60.0 / 60.0 / 24.0
 
-      should_have_needs_attention_label = days_since_updated > UNTOUCHED_PR_DAYS
+      should_have_needs_attention_label = days_since_created > NEEDS_ATTENTION_PR_LIFESPAN_DAYS
       has_needs_attention_label = has_label?(pr, NEEDS_ATTENTION)
 
-      if should_have_needs_attention_label
+      if should_have_needs_attention_label || has_needs_attention_label
```

Comment:
Added the "or" last minute so that things we've manually applied the label to will get counted as part of the Slack report